README.md

```diff
@@ -1,3 +1,12 @@
 # rolldown
 
 [![Travis build status](https://travis-ci.com/yihui/rolldown.svg?branch=master)](https://travis-ci.com/yihui/rolldown)
+![](https://img.shields.io/badge/lifecycle-experimental-orange.svg)
+
+An extension to [R Markdown](https://rmarkdown.rstudio.com/) that builds scroll-driven HTML documents for storytelling, a.k.a _scrollytelling_, powered by JavaScript libraries. Try here:
+```r
```

Comment:
Do you remember that I bowed to one presenter with apologies for nitpicking the insufficient spaces and empty lines at the unconf?

Please add an empty line before this line. Reasons: https://yihui.name/cn/2017/05/blank-line/

Reply:
I actually overlooked this when I threw together the draft. 
Normally when I use Markdown, I like to put two empty lines prior to a level 1 header, and one empty line for the rest of the headers and other elements such as code blocks. I hope this is OK.

Reply:
Sure.